### ISSUE TITLE
x86 32bit image

### ISSUE THREAD

**Issue by shazow** [jpadilla/juicebox#30]
Is it feasible to make a 32bit build of the Juicebox guest image? Looks like boxcutter has them.

Ran into [a problem here](http://discuss.hellowebapp.com/t/juicebox-seeing-hardware-acceleration-not-available-error-on-start-is-this-ok/112) where some devices don't have a CPU that supports the necessary instructionset to run 64-bit guest machines.

**Comment by jpadilla** [comment]
@shazow hmm you might actually be able to enable virtualization through the BIOS. I think @sezalcru ran into this a while back too.

**Comment by cesar** [comment]
That is correct, enabling "Set Virtualization" from the BIOS should do the trick.

**Comment by shazow** [comment]
Thanks for the suggestion! I added a reply mentioning that.

If 32bit images are out of scope for the project, feel free to close the issue. If you have any pointers for how to build them myself, I might give it a try. :)

**Comment by jpadilla** [comment]
@shazow I'm giving it a try but slow internet. You could download this [template](https://gist.github.com/jpadilla/60ae764f0b452da4409b) and try running:

```
$ packer build desktop-i386.json
```

**Comment by jpadilla** [comment]
@shazow seems to be working for me. Unfortunately docker won't work on 32bit.

**Comment by jpadilla** [comment]
@shazow just pushed the working templates to this repo. Won't probably publish an image just yet, but feel free to do so yourself.

**Comment by shazow** [comment]
@jpadilla Thanks!
